coral/src/domain/schema-request/schema-request-api.ts

```diff
@@ -58,18 +58,34 @@ const getSchemaRequestsForApprover = (
     .then(transformGetSchemaRequestsForApproverResponse);
 };
 
-const approveSchemaRequest = (payload: RequestVerdictApproval<"SCHEMA">) => {
+type ApproveSchemaRequestPayload = ResolveIntersectionTypes<
+  Omit<RequestVerdictApproval<"SCHEMA">, "requestEntityType">
+>;
+const approveSchemaRequest = (payload: ApproveSchemaRequestPayload) => {
+  const payloadPassed: RequestVerdictApproval<"SCHEMA"> = {
+    ...payload,
+    requestEntityType: "SCHEMA",
+  };
+
   return api.post<
     KlawApiResponse<"approveRequest">,
     RequestVerdictApproval<"SCHEMA">
-  >(`/request/approve`, payload);
+  >(`/request/approve`, payloadPassed);
 };
 
-const declineSchemaRequest = (payload: RequestVerdictDecline<"SCHEMA">) => {
+type DeclineSchemaRequestPayload = ResolveIntersectionTypes<
+  Omit<RequestVerdictDecline<"SCHEMA">, "requestEntityType">
+>;
+const declineSchemaRequest = (payload: DeclineSchemaRequestPayload) => {
+  const payloadPassed: RequestVerdictDecline<"SCHEMA"> = {
+    ...payload,
+    reqIds: ["4325252524"],
```

Comment:
Unsure about these default values passed here, and also unsure about the need for `payloadPassed` to exist (here and in `approveSchemaRequest` ? Isn't `RequestVerdictApproval<"SCHEMA">` enough?